src/TCommandLine.cpp

```diff
@@ -35,12 +35,12 @@
 #include "post_guard.h"
 
 
-TCommandLine::TCommandLine( Host * pHost, TConsole * pConsole, QWidget * parent )
-: QPlainTextEdit( parent )
-, mpHost( pHost )
-, mpConsole( pConsole )
-, mSelectedText( "" )
-, mSelectionStart( 0 )
+TCommandLine::TCommandLine(Host* pHost, TConsole* pConsole, QWidget* parent)
+: QPlainTextEdit(parent)
+, mpHost(pHost)
+, mpConsole(pConsole)
+, mSelectedText("")
```

Comment:
Isn't the `""` surplus to requirements here - and if it is a `QString` this is not ideal either {will trip up on future `#define QT_NO_CAST_{TO|FROM}_ASCII`}... :frowning_face: